configs/opa_bundles/default/authz.rego

```diff
@@ -46,13 +46,28 @@ allow if {
 	input.user_id == input.resource_id
 }
 
+allow if {
+	authn
+	input.kind == "usr"
+	input.action == "get-org-member-status"
+	member_of_single_assosicated_org_id
+}
+
+# anyone can translate an email to an id.
```

Comment:
I don't agree with this rule, let's discuss it offline.

Reply:
discussed offline.